models/marts/core/fct_hospital_events.yml

```
@@ -0,0 +1,37 @@
+version: 2
+
+models:
+  - name: fct_hospital_events
+    description: '{{ doc("fct_hospital_events") }}'
+    columns:
+      - name: hospital_sk
+        description: '{{ doc("hospital_sk") }}'
```

```suggestion
        description: A primary key of the fct_hospital_events table, composed of `encounter_id` and `procedure_id`.
```

I think you can probably just put the description here and not in doc blocks, I don't know when `hospital_sk` would show up in another table. This helps to make doc blocks smaller and to make the description immediately accessible.

Also, is there a need to have a surrogate key rather than just using `encounter_id`? It doesn't look like there's any encounters where there are multiple procedures.
```
SELECT
  encounters.encounter_id,
  count(*)
FROM `datatonic-americas-demos.dbt_cchow.stg_encounters` as encounters
left join `datatonic-americas-demos.dbt_cchow.stg_procedures` as procedures on encounters.encounter_id = procedures.procedure_id 
group by 1 
order by 2 desc 
limit 1000
```

---

I agree with the description. I will make this change shortly.
As far as the second part - I think you are joining on the wrong key.
It should be `encounters.encounter_id = procedures.encounter_id` instead of `encounters.encounter_id = procedures.procedure_id`

---

Oops, you're totally right, I joined on the wrong key, SK makes sense to have!